internal/testutils/balancer.go

```diff
@@ -339,6 +339,14 @@ func IsRoundRobin(want []balancer.SubConn, f func() balancer.SubConn) error {
 	return nil
 }
 
+// SubConnFromPicker creates a new SubConn by taking in the Picker to use.
```

Comment:
Consider modifying the comment to something like this, since this function technically doesn't create a new subConn. It just returns the subConn returned by the picker. And the picker also doesn't create a new subConn, it simply returns **a** subConn (from the list of existing subConns) to service this RPC.

```golang
// SubConnFromPicker returns a function which returns a SubConn by calling the Pick()
// method of the provided picker. There is no caching of SubConns here. Every invocation
// of the returned function results in a new pick.
```

Reply:
done. thanks for that!